pkg/clusteroperator/list.go

```diff
@@ -0,0 +1,121 @@
+package clusteroperator
+
+import (
+	"context"
+	"time"
+
+	"github.com/golang/glog"
+	"github.com/openshift-kni/eco-goinfra/pkg/clients"
+	metaV1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+)
+
+// List returns clusterOperators inventory.
+func List(apiClient *clients.Settings) ([]*Builder, error) {
```

Comment:
Do we want to rename List to ClusterOperatorList ?

Reply:
nope